Revision Tests/maryna_boieva.md

```diff
@@ -0,0 +1,31 @@
+## Please translate into English
+
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If only I were more clever, I would have made a backup of the database yesterday and only after that apply my PR to production. I wish, i were as clever as my father.
+
+**_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
+
+> If all the students had read the theory and only after that wrote a translation analysing the sentences, the progress would be obvious. If only every one didn't tried to learn English brute force, escaping the early stages of analisis and skills training. I wish you didn't hurry up.
+
+**_3. Если бы сегодня мне не нужно было садиться за руль, то вчера бы я знатно набрался с друзьями в ближайшем баре и орал бы в микрофон свою любимую песню. Жаль что автопилот не является повсеместной функцией во всех автомобилях, это могло бы значительно разнообразить досуг водителей._**
+
+> If I hadn't had to drive today, I would have got drunk in the nearest bar with friends yesterday and screamed my favourite song in the microphone. If with the authopilot was a common function in all of the cars. It could greatly vary the drivers free time.
+
+**_4. Если бы она проверяла работоспособность билда после слияния своих коммитов с мастером в прошлом, то сейчас бы у нее не было бы общения с клиентом по поводу даунтайма на проекте и финансовых потерях в целом._**
+
+> If she checked the workability of the build after merging her commits with master, she wouldn't have had a conversation with the client about the downtime on the project and financial losses in general.
```

Comment:
If she had checked (in the past, please be more careful with Russian)
She wouldn't have the conversation with the client about (there is a description of the topic for discussion)